backend/curriculum_tracking/api_views.py

```diff
@@ -883,6 +891,26 @@ def auto_assign_reviewers(self, request, pk=None):
             response = actor.send()
             return Response({"status": "OK", "data": response.asdict()})
 
+    @action(
+        detail=False,
+        methods=["POST"],
+        serializer_class=serializers.SetDueTimeSerializer,
+        permission_classes=[HasObjectPermission(permissions=Team.PERMISSION_MANAGE_CARDS)]
+    )
+    def bulk_set_due_dates(self, request, pk=None):
+        serializer = self.get_serializer(data=request.data)
+        if serializer.is_valid():
+            teams = self.get_object()   # This will get you all the teams, we only need the teams in the request data
```

Comment:
There should only be one team.

If we make this an action on the TeamVIewSet and have `detail=True` then the loop wont be needed.

Reply:
i tried setting detail=True but the result is that it removes the option to bulk set due dates from the @actions list.  I found a work around though.